Sources/NIO/ByteBuffer-aux.swift

```diff
@@ -169,6 +171,79 @@ extension ByteBuffer {
         return self.getString(at: self.readerIndex, length: length)! /* must work, enough readable bytes */
     }
 
+    // MARK: DispatchData APIs
+    /// Write `dispatchData` into this `ByteBuffer`, moving the writer index forward appropriately.
+    ///
+    /// - parameters:
+    ///     - dispatchData: The `DispatchData` instance to write to the `ByteBuffer`.
+    /// - returns: The number of bytes written.
+    @discardableResult
+    public mutating func write(dispatchData: DispatchData) -> Int {
```

Comment:
Any reason this isn't `@inlinable`?

Reply:
@Lukasa don't think there would be any benefit there. No closures/generics so a DispatchData is just a pointer (wrapped in a struct) and we return just an Int.

Reply:
In which case, why is `set` inlinable? I don’t really care either way, but I do think we should be consistent.

Reply:
@Lukasa oh, sorry, I'll remove that. Too much c&p